### PR TITLE
Fixing setup script

### PR DESCRIPTION
**Pull request by mtsadler**
Updated:
- CircleCI script to work the same as other plugins
- Marked certain tests as compatibility tests
- Renamed files to follow plugin template
- Moved test querying to at runtime (declared in views instead of forms)
- Added timezone as hiddenfield for non-rbac

### CODE REVIEW

File: bin/setup [hunk 1]
```diff
@@ -157,15 +168,15 @@ if __name__ == "__main__":
 
     parser = argparse.ArgumentParser(
         description="""
-    rb status is a plugin for Airflow that handles data confidence.  To operate, it needs
```
Comment (kmcmanus) [inline bin/setup:160]: I feel like we want to keep _this_ line.

File: core/report_model.py [hunk 2]
```diff
@@ -174,7 +175,7 @@ class ReportForm(Form):
                     "List of the tests to include in the report. Only includes\
                  tasks that have ran in airflow."
                 ),
-                choices=get_all_test_choices(),
```
Comment (kmcmanus) [inline core/report_model.py:177]: Having removed this _here_ don't we want to set it in the views?

Comment (mtsadler) [reply core/report_model.py:177]: `get_all_test_choices` was also being called in ReportModel's `create_form` and `edit_form`, which are the only times we need to generate a list of all the tests.

Comment (mtsadler) [reply core/report_model.py:177]: So the functionality was already there, and we could just remove that line.

**Pull request merged**